data/fields/cuisine.json

```
@@ -16,16 +16,24 @@
             "kebab": "Kebab",
             "indian": "Indian",
             "chicken": "Chicken",
+            "fish": "Fish",
             "asian": "Asian",
             "french": "French",
+            "spanish": "Spanish",
+            "portugese": "Portugese",
```

Fix the typo Codespell found:
```suggestion
            "portuguese": "Portuguese",
```